examples/colors_rgb.rs

```diff
@@ -37,16 +37,12 @@ use palette::{convert::FromColorUnclamped, Okhsv, Srgb};
 use ratatui::{
     backend::{Backend, CrosstermBackend},
     buffer::Buffer,
-    crossterm::{
-        event::{self, Event, KeyCode, KeyEventKind},
-        terminal::{disable_raw_mode, enable_raw_mode, EnterAlternateScreen, LeaveAlternateScreen},
-        ExecutableCommand,
-    },
+    crossterm::event::{self, Event, KeyCode, KeyEventKind},
     layout::{Constraint, Layout, Rect},
     style::Color,
-    terminal::Terminal,
     text::Text,
     widgets::Widget,
+    Terminal,
```

Comment:
I don't think I like that this kind of change is possible in the first place… would be breaking, but the public export should only exist in a single location. Either it's something global or it needs structuring in modules. Allowing for both seems strange. General thought here: reduce the stuff that is publicly accessible. One of the two places should be gone. (Unrelated to this PR, should be done in its own PR.)

Reply:
Yeah agreed., and irrelevant to this PR